mantidproject/mantid
# ILL calibration tab

Fixes [#11163](http://trac.mantidproject.org/mantid/ticket/11163).

To test:
- Review docs
- Set facility to ILL
- Open Indirect Data Reduction > Energy Transfer
- Load `ILLIN16B_034745.nxs` from unit test data
- Select File grouping, load `Babylon5/Public/DanNixon/IN16B/IN16B_PSD_Only.xml`
- Enable Mirror Mode
- Run (this gives an energy transfer without calibration to compare to, plot the colour fill of it)
- Go to Calibration tab and enter same run and grouping, enable mirror mode
- Run
- Go back to Energy Transfer and specify the calibration workspace just created
- Run, plot the colour fill plot

Notice that the peak in the calibrated data is a very neat bar across all Q around the elastic region (compare to the original data), when using the same run in calibration and reduction this shows that the calibration is being applied correctly.

The PSD only mask is required to normalise the calibration correctly.
